lstchain/reco/utils.py

```diff
@@ -862,3 +863,51 @@ def get_events_in_GTI(events, CatB_cal_table):
     gti_mask = gti[events['calibration_id']]
 
     return events[gti_mask]
+
+def compute_rf_event_weights(events):
+    """
+    Compute event-wise weights. Can be used for correcting for the different
+    statistics present in each pointing node of the MC training sample,
+    to avoid "jumps" in the performance of the random forests
+
+    events: a DL1 parameters dataframe
+    """
+
+    # Add a 'weight' column to the input table
+    weights = np.array(np.ones(len(events)))
+
+    # First identify existing telescope pointings in the sample:
+    # Convert to degrees and round to avoid potential
+    # rounding issues:
+    alt = np.round(events['alt_tel'], decimals=5)
+    az = np.round(events['az_tel'], decimals=5)
+    dummy = np.unique(np.array([alt, az]).T, axis=0)
```

Comment:
`unique` already returns sorted values, so if you want to have it sorted by azimuth, you can just do `np.unique([alt, az], axis=1)`

Reply:
No, it does not work... behavior changed, I don't know why